packages/ui/primitives/document-flow/missing-signature-field-dialog.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import { Button } from '@documenso/ui/primitives/button';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from '@documenso/ui/primitives/dialog';
+
+export type MissingSignatureFieldDialogProps = {
+  isOpen: boolean;
+  onOpenChange: () => void;
+};
+
+export const MissingSignatureFieldDialog = ({
+  isOpen,
+  onOpenChange,
+}: MissingSignatureFieldDialogProps) => {
+  return (
+    <Dialog open={isOpen} onOpenChange={onOpenChange}>
+      <DialogContent className="max-w-lg" position="center">
+        <DialogHeader>
+          <DialogTitle>No signature field found</DialogTitle>
+          <DialogDescription>
+            <p className="mt-2">
+              Some signers have not been assigned a signature field. Please assign a signature field
+              to each signer before proceeding.
+            </p>
+          </DialogDescription>
+        </DialogHeader>
+        <DialogFooter>
+          <Button type="button" variant="secondary" onClick={onOpenChange}>
+            Cancel
+          </Button>
+          <Button type="submit" onClick={onOpenChange}>
+            Proceed
```

Comment:
Do we still need both options if you can't proceed? Why the 2 buttons?

Reply:
True that. I kept it like this for consistency. Should I just go ahead and remove the proceed button?

Reply:
- Since this is not just a hint why you can't contine it can be a alert actually